renovate.json

```diff
@@ -29,6 +29,14 @@
       "matchStrings": ["python-version:\\s(?<currentValue>.*)"],
       "datasourceTemplate": "github-tags",
       "depNameTemplate": "python/cpython"
+    },
```

Comment:
This should be available as preset by now.

Reply:
https://docs.renovatebot.com/presets-regexManagers/#regexmanagersdockerfileversions

Reply:
Oh, you are right, this is nice! 🚀 
Replaced it with `regexManagers:dockerfileVersions` (https://docs.renovatebot.com/presets-regexManagers/#regexmanagersdockerfileversions).